### ISSUE TITLE
Clean install of react-editor example throws uncaught reference error

### ISSUE THREAD

**Issue by brianlovin** [yuanqing/create-figma-plugin#125]
I just did a clean install of the react-editor example plugin, and on first-run I get this in Figma:

<img width="777" alt="Screen Shot 2022-10-07 at 14 15 26@2x" src="https://user-images.githubusercontent.com/1923260/194653553-50b9e00f-6ef5-4cca-88b3-69f3851ca0dc.png">

Repro steps:

1. `npx --yes create-figma-plugin`
2. `cd create-figma-plugin`
3. `npm i`
4. `npm run build`
5. `npm run watch`
6. Add `manifest.json` to Figma
7. Run plugin

**Comment by yuanqing** [comment]
Thanks! I’m able to reproduce this.

It seems like the opening two lines from the file `src/ui.tsx` in the `plugin/react-editor` template weren’t actually needed and can be removed:

```tsx
import 'prismjs/components/prism-clike.js'
import 'prismjs/components/prism-javascript.js'
```

The plugin should work normally without the errors with these lines removed.

**Comment by yuanqing** [comment]
Fixed in `2.1.4`
